tests/NetworkTest.py

```diff
@@ -32,15 +32,16 @@ def test_aneuron(self):
         """
         self.assertTrue(isinstance(self.net.aneuron('AVAL'),PyOpenWorm.Neuron))
 
+    @unittest.expectedFailure
```

Comment:
Well, the changes in this test were only caused by trying to solve #178 - I think this PR can be closed and not merged, actually. I don't see anything useful in it other than the observations it allowed us to make.
With the new information, we can try to see what we should do for #178 , change the wording for the original `test_neurons`, and I can fix #228 separately without breaking anything. :smile: